### PR TITLE
OTOW-5667 Fixed the admin.css throwing 404 error

### PR DESCRIPTION
**Pull request by rapbhan**

### CODE REVIEW

**Comment by rapbhan** [comment]
retest this please

**Comment by PDegenPortnoy** [comment]
Does moving the administration CSS into a subdirectory prevent it from being loaded with non-administrative pages? If so, then this looks good, but I just wanted to confirm that interpretation

**Comment by rapbhan** [comment]
You're right @PDegenPortnoy!

**Comment by rapbhan** [comment]
We'll deploy it to staging and retest it before merging!

File: app/views/layouts/admin.html.haml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     %meta{ charset: 'utf-8' }
     %title OpenHub Admin
     %meta{ content: 'width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no', name: 'viewport' }
-    = stylesheet_link_tag 'admin', media: 'all'
+    = stylesheet_link_tag 'admin/admin', media: 'all'
     = javascript_include_tag 'application'
     %link{ href: '//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,300italic,400italic,600italic' }
   %body.hold-transition.skin-blue.sidebar-mini
```
Comment (TOP3MONOPOLY) [inline app/views/layouts/admin.html.haml]: Clearance.configure do |config|

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -3,5 +3,5 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
-Rails.application.config.assets.precompile += %w[permissions.js admin.css]
+Rails.application.config.assets.precompile += %w[permissions.js admin/admin.css]
```
Comment (TOP3MONOPOLY) [inline config/initializers/assets.rb]: Clearance.configure do |config|

File: app/views/layouts/admin.html.haml [hunk 3]
```diff
@@ -5,7 +5,7 @@
     %meta{ charset: 'utf-8' }
     %title OpenHub Admin
     %meta{ content: 'width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no', name: 'viewport' }
```
Comment (TOP3MONOPOLY) [inline app/views/layouts/admin.html.haml]: Clearance.configure do |config|

File: app/views/layouts/admin.html.haml (hunk 1 above)
Comment (TOP3MONOPOLY) [inline app/views/layouts/admin.html.haml]: Clearance.configure do |config|

File: config/initializers/assets.rb [hunk 4]
```diff
@@ -3,5 +3,5 @@
...
 Rails.application.config.assets.precompile << /\.(?:svg|eot|woff|ttf)\z/
```
Comment (TOP3MONOPOLY) [inline config/initializers/assets.rb]: Clearance.configure do |config|

**Pull request merged**